tests/test_bs.py

```diff
@@ -381,6 +382,25 @@ def save_log_on_exit(request):
         (out, err) = p.communicate()
 
 
+@pytest.fixture()
+def add_curl(request, setup_bootstrap):
```

Comment:
we have too many fixtures at the begining of each test, consider making this in higher level so it won't need to be specified each time

Reply:
I'm not sure I understand this comment.
Fixtures have order. each fixture depends on other fixtures.
To deploy full network you need to call only 1 fixture - setup_clients --> this fixture will call other fixtures as needed.